server/issue.go

```diff
@@ -819,3 +819,31 @@ func attachFileToIssue(api plugin.API, jiraClient *jira.Client, issueKey, fileId
 	attachment := (*attachments)[0]
 	return fileinfo.Name, attachment.Filename, nil
 }
+
+func JiraGet(jiraClient *jira.Client, api string, params map[string]string, dest interface{}) (int, error) {
+	return jiraGet(jiraClient, 2, api, params, dest)
+}
+
+func JiraGet2(jiraClient *jira.Client, api string, params map[string]string, dest interface{}) (int, error) {
```

Comment:
Unused function?

Reply:
Note, my linter warns me of dead code. Should we hook that sort of linting into CI?